### PR TITLE
Update some TODOs

### PR DESCRIPTION
**Pull request by AlekSi**
## Readiness checklist

* [ ] I added tests for new functionality or bugfixes.
* [x] I ran `task all`, and it passed.
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] I ensured that the title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: integration/update_field_compat_test.go [hunk 1]
```diff
@@ -98,7 +98,7 @@ func TestUpdateFieldCompatMax(t *testing.T) {
 		},
 		"DuplicateQuery": {
 			update: bson.D{{"$max", bson.D{{"v", int32(39)}, {"v", int32(30)}}}},
-			skip:   "Handle duplicates correctly",
+			skip:   "https://github.com/FerretDB/FerretDB/issues/666",
```
Comment (rumyantseva) [inline integration/update_field_compat_test.go:101]: That's covered in https://github.com/FerretDB/FerretDB/pull/1391/files

**Review by rumyantseva — COMMENTED** [review]
LGTM overall, one thing is already covered in https://github.com/FerretDB/FerretDB/pull/1391/files and doesn't need to be mentioned here.

**Review by chilagrow — DISMISSED** [review]

**Review by w84thesun — DISMISSED** [review]

**Review by noisersup — COMMENTED** [review]
There are merge conflicts

**Review by w84thesun — APPROVED** [review]

**Review by rumyantseva — APPROVED** [review]
LGTM

**Pull request merged**